src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -139,15 +169,7 @@
     private int topCitizenId = 0;
     private int maxCitizens  = Configurations.maxCitizens;
 
-    /**
-     * The Positions which players can freely interact.
-     */
-    private final Set<BlockPos> freePositions = new HashSet<>();
-
-    /**
-     * The Blocks which players can freely interact with.
-     */
-    private final Set<Block> freeBlocks = new HashSet<>();
+    private double overallHappiness = 5;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)